addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14266,13 +14276,13 @@ msgstr ""
 #. Description of setting "Videos -> Library -> Flatten TV show seasons" with label #20412
 #: system/settings/settings.xml
 msgctxt "#36144"
-msgid "Remove the TV show season node, toggles between 'If only one season' (default), 'Always' and 'Never'."
+msgid "Remove the TV show season node, toggles between \"If only one season\" (default), \"Always\" and \"Never\"."
```

Comment:
sure about what? That the words in quotes are values that can be selected for this setting? Yes, that I'm sure about.
But honestly, the reference is IMO not needed here at all as they are quite self explaining. Also this setting is no longer toggling as it's no spinner anymore. Maybe rewrite it so something like:
```
Select in which cases you want to hide the season nodes of TV shows. If hidden, selecting a TV show will directly jump into the episodes view.
```